1.preprocessing-data/ICutils.py

```diff
@@ -73,15 +81,16 @@ def run_illum_correct(
     """Calculates flatfield, darkfield, performs illumination correction on channel images, coverts to 8-bit and saves images into designated folder
 
     Parameters:
-        channels_path (pathlib.Path): Path to directory where the all images for each channel are stored
+        data_path (pathlib.Path): Path to directory where the folders for each plate with data are stored
         save_path (pathlib.Path): Path to directory where the corrected images will be saved to
+        plate (str): Name of plate
         channel (str): Name of channel
         output_calc (bool): Outputs plots of the flatfield and darkfield function for visual review if set to 'True' (default = False)
         file_extension (str): Sets the file_extension for the images
         overwrite (bool): Will save over existing images if set to 'True' (default = False)
     """
     # Loads in the variables returned from "load_pybasic_data" function
-    images, image_names = load_pybasic_data(channel_path, channel, file_extension)
+    images, image_names = load_pybasic_data(data_path, plate, channel, file_extension)
```

Comment:
I tend to prefer named arguments rather than positional arguments for function calling (especially for custom functions).

In other words, this function call would become:

```python
images, image_names = load_pybasic_data(
    data_path=data_path,
    plate=plate,
    channel=channel,
    file_extension=file_extension
)
```

By using named arguments, we ensure changes like the one in line 93 (that introduce new function arguments) are not breaking.

Just my 2 cents!

Reply:
Easy fix! I was struggling with things not running because I didn't have all the arguments.